pkg/kubelet/server/server_test.go

```diff
@@ -1398,7 +1409,9 @@ func TestServePortForward(t *testing.T) {
 				c = &http.Client{Transport: upgradeRoundTripper}
 			}
 
-			resp, err := c.Post(url, "", nil)
+			req := makeReq(t, "POST", url)
+			req.Header.Add("X-Stream-Protocol-Version", "portforward.k8s.io")
```

Comment:
is this intending to add a header so that the client claims to support both `v4.channel.k8s.io` and `portforward.k8s.io`?

Reply:
I saw that (and that it's set to v4 in `makeReq`) but didn't comment on it because it was in the test. The more appropriate thing is probably to adjust `makeReq` to take in the header value as an additional function parameter?

Reply:
Yes - both are supported.

Reply:
> both are supported.

not for port forwarding they're not